GraphicalVersion/Snapshot.cs

```diff
@@ -102,10 +102,12 @@ public static string getUncompressedSnapshot(string compressedSnapshot)
         public static bool convertToPDF(string uncompressedSnapshotName)
         {
             string outputPDF = Path.GetFileNameWithoutExtension(uncompressedSnapshotName) + ".pdf";
+            string dummyPDF = Path.GetFileNameWithoutExtension(uncompressedSnapshotName) + "_dummy.pdf";
             bool r = false;
             try
             {
                 r = ConvertUncompressedSnapshot(uncompressedSnapshotName, outputPDF, 0, "", "", 0, 0, 0);
+                r = MergePDFDocuments(outputPDF, dummyPDF);

```

Comment:
Suggest commenting briefly here that the merge with dummy pdf file which does not exist is to remove printing file restrictions.